src/bundle/Resources/config/services/controllers.yaml

```diff
@@ -28,6 +28,7 @@ services:
 
     Ibexa\Bundle\User\Controller\UserRegisterController:
         autowire: true
+        autoconfigure: true
```

Comment:
Considering other controllers in the file did not receive this change, and that this tags were already applied to the service declaration, what is the difference? What in particular was missing (not autoconfigured)?